### ISSUE TITLE
Add a way to force an instance to use a particular language

### ISSUE THREAD

**Issue by bpeel** [bookwyrm-social/bookwyrm#2777]
**Describe the solution you'd like**

If an instance is targeting a particular language community, I think it would be nice to be able to force the site to be in that language. Of course the user can choose what language they want to display in via the settings, but it would be nice to have this default language even during the account creation process.

**Describe alternatives you've considered**

I’m not exactly sure how BookWyrm picks the language before the user configures their settings, but I suspect that it looks at the browser’s language settings because when I visit bookwyrm.social without an account it defaults to French, which is what my browser is configured to prefer.

There is a `LANGUAGE_CODE` setting in `.env` which claims to set the default language, but this doesn’t seem to affect anything, so maybe it doesn’t have priority over the user’s browser settings.

**Additional context**

This would be particularly useful in my case because I am trying to make an instance for the Esperanto-speaking community. Probably very few people would configure their browser to prefer Esperanto but I’m sure most of the future users would prefer to have the site in Esperanto.

**Comment by WesleyAC** [comment]
I agree that better settings around this would be good, but as a temporary solution, you can comment out (prefix with `#`) or delete the line `"django.middleware.locale.LocaleMiddleware",` in `bookwyrm/settings.py` in the `MIDDLEWARE` section, and then the instance will show up in the configured language for all users, regardless of user or browser setting.

**Comment by bpeel** [comment]
> you can comment out (prefix with `#`) or delete the line `"django.middleware.locale.LocaleMiddleware",` in `bookwyrm/settings.py`

Oh brilliant, that works really well. Thanks!

**Comment by mouse-reeve** [comment]
I'm closing this issue because it's very old, and the codebase has changes substantially in the subsequent years.
